assets/php/virion.php

```diff
@@ -98,7 +108,11 @@ function virion_infect(Phar $virus, Phar $host, string $prefix = "", int $mode =
     }
```

Comment:
An antibody compatible with src-namespace-prefix is also needed here

Reply:
My bad, it's line 106

Reply:
Not here, virions don't support PSR-4

Reply:
But without this, the plugin cannot find the virion classes

Reply:
What?
I don’t think you understand what this bits doing….
Regardless of PSR-0/PSR-4 shading the virion code with a prefix will always be the same…